src/main/java/org/folio/inventory/resources/ItemRepresentation.java

```diff
@@ -166,6 +165,10 @@ private JsonObject toJson(
       log.warn(String.format("Failed to create self link for item: %s", e.toString()));
     }
 
+    if (item.getLastCheckIn() != null) {
```

Comment:
I think that it will better to use "Objects.nonNull()" here. What do you think?

Reply:
Why?

Reply:
API Note: This method exists to be used as a Predicate, filter(Objects::isNull)

Reply:
I think that we can avoid here NullPointerException if 'item' will be null in this way.

Reply:
Oh, really. Thank you!